ynput/ayon-core
# Chore: Use 'folderPath' instead of 'asset' during publishing

## Changelog Description
Rename `"asset"` key used during publishing to `"folderPath"`. This is first of many PRs changing asset naming to folder.

## Additional info
I think resolve, hiero and traypublisher might not be converted correctly. Please @jakubjezek001 test them and if you find and issue try to fix it.

## Testing notes:
Ideal test scenario:
1. Open a host with workfile that already did have created instances.
2. Use all possible creators.
3. Publish all the instances.
4. Everything is working and shiny.
- [x] 3dsMax
- [x] SubstancePainter
- [x] TrayPublisher

I do expect that hosts using Publisher tool should not have any issues. Possible issues are with hosts using pyblish pype, automated publishin login and editorial.
